### PR TITLE
https://eddvq.github.io/html200-adopt-a-dog/

### PR DESCRIPTION
**Pull request by eddvq**

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        <ul class="header-nav">
+          <li><a href="index.html">Home</a></li>
+          <li><a href="our-dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </div>
+    </header>
+    <main>
+      <h1>Adoptadog Blog</h1>
+      <article>
```
Comment (tylervanbrocklin) [inline blog.html]: You'll want to create a CSS class to style these articles. So the text is beside the image. See PDF for example.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      <div>
+        <ul class="header-nav">
+          <li><a href="index.html">Home</a></li>
+          <li><a href="our-dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </div>
+    </header>
+    <main>
+      <section>
```
Comment (tylervanbrocklin) [inline index.html]: You will want to wrap your dog tiles in div tags, not sections. You might use a section to wrap all of your dog-tiles into a container, but div tags should be used for the individual tiles. Also, you'll want to put some styling in. When I visit your site the dog images take up the entire page. See if you can add a css class to these dog tiles to bring them down in size. Then wrap these dog tiles in a section or div, and apply a css class to that to create the grid appearance.

File: blog.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name= "viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <link rel="stylesheet" type="text/css" href="styles/normalize.css">
```
Comment (jimmynono) [inline blog.html:7]: You'll generally want to put utility css above the custom css you write. So ti doesn't overwrite what you wrote. Order matters with a lot of code

File: styles/styles.css [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+.footer-elements p {
+	font-family: "Open sans", sans-serif;
+	font-size: 1rem;
+	width: 60%;
+	color: white;
+	margin-bottom: 0;
+	text-align: center;
+}
+
+small {
```
Comment (jimmynono) [inline styles/styles.css:174]: this element name doesn't quite reflect what it does. I'd give it a class that's descriptive of what it is,  something like color-white

File: styles/styles.css [hunk 5]
```diff
@@ -0,0 +1,368 @@
...
+}
+
+
+/* INDEX CSS */
+
+main {
+	margin: 1rem 10rem;
+}
+
+.about-us {
```
Comment (jimmynono) [inline styles/styles.css:185]: the dog image needs to be to the left of the text on desktop views

File: styles/styles.css [hunk 6]
```diff
@@ -0,0 +1,368 @@
...
+.about-us-blurb p {
+	font-size: 1.4rem;
+}
+
+
+/* DOG CARD CSS */
+
+.dog-card-board {
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
```
Comment (jimmynono) [inline styles/styles.css:214]: YES! This code is the best. Magic.

Comment (jimmynono) [reply styles/styles.css:214]: maybe translate the px to rems though

File: styles/styles.css [hunk 7]
```diff
@@ -0,0 +1,368 @@
...
+.dog-card>h3 {
+	margin-left: 1rem;
+}
+
+.dog-card-details>p:nth-of-type(1) {
+	display: inline;
+	font-weight: 700;
+}
+
+.dog-card-details>p:nth-of-type(2) {
```
Comment (jimmynono) [inline styles/styles.css:246]: these n-th of type might get complicated fast. Maybe instead use a class for that element

File: styles/styles.css [hunk 8]
```diff
@@ -0,0 +1,368 @@
...
+}
+
+.dog-card-details>button {
+	background-color: green;
+	font-size: 1rem;
+	font-family: "Open sans", sans-serif;
+	font-weight: 700;
+	color: white;
+	width: 100%;
+	padding-top: 10px;
```
Comment (jimmynono) [inline styles/styles.css:259]: translating these to rems woudl be good so they scale along with the fonts

File: styles/styles.css [hunk 9]
```diff
@@ -0,0 +1,390 @@
...
+		margin: 2rem auto 0 auto;
+	}
+	.header-top {
+		margin: 0 1.5rem 1.25rem 1.5rem;
+	}
+	/* Full Span Button on Mobile */
+	.shopping-cart {
+		flex: 100%;
+	}
+	.cart-button {
```
Comment (jimmynono) [inline styles/styles.css:343]: inside media queries you'll only want to put in what you're changing. Most of this is unneeded in the media query. It'll make maintaining things easier.

File: styles/styles.css [hunk 10]
```diff
@@ -0,0 +1,390 @@
...
+	.about-us-blurb p {
+		font-size: 1.2rem;
+	}
+	.blog-container {
+		grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
+		gap: 2rem;
+	}
+}
+
+@media only screen and (max-width: 850px) {
```
Comment (jimmynono) [inline styles/styles.css:319]: technically I should be able to see the blog container at ipad with per the assignment. You could adjust this value to make that happen.

File: blog.html [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+    <header>
+      <div class="header-top">
+        <div class="logo-bundle">
+          <img class="header-logo" src="images/adoptadog-logo.svg" alt="Adopt a Dog Logo">
+          <p>Seattle's premiere dog adoption agency</p>
+        </div>
+        <div class="shopping-cart">
+          <img class="shopping-cart-icon" src="images/cart-icon.svg" alt="Shopping cart icon">
+          <p>$1234.56</p>
+          <button class="cart-button" type="button">Checkout</button>
```
Comment (tylervanbrocklin) [inline blog.html]: appears to be no way to get to the checkout page. need to add a link here

File: blog.html [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <link rel="stylesheet" type="text/css" href="styles/normalize.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Lora:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <title>Adopt a Dog - Blog</title>
+  </head>
+  <body>
+    <header>
+      <div class="header-top">
+        <div class="logo-bundle">
```
Comment (tylervanbrocklin) [inline blog.html:15]: nice work adding the gradient!

File: checkout.js [hunk 13]
```diff
@@ -0,0 +1,210 @@
...
+
+//This is where the Dog cards are assembled
+
+const dogQuestion = document.createElement('p');
+dogQuestion.innerText = 'Dogs Adopted';
+
+form.appendChild(dogQuestion);
+
+for(let i = 0; i < dogHouse.length; i++){
+  const dogCard = document.createElement('img');
```
Comment (tylervanbrocklin) [inline checkout.js:148]: make sure to add styling to these dogs. The image takes up the entire screen on desktop

**Pull request closed without merging**